qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -1044,7 +985,7 @@ def to_dataframe(self):
             meta = qdb.sql_connection.TRN.execute_fetchindex()
 
             # Create the dataframe and clean it up a bit
-            df = pd.DataFrame((list(x) for x in meta), columns=cols)
+            df = pd.DataFrame((list(x) for x in meta), columns=cols, dtype=str)
```

Comment:
I guess that's possible if something in the database is NULL but if it is, my guess is that it's fine ...